src/Console/Command/DispatchCommand.php

```diff
@@ -45,13 +64,94 @@ protected function configure()
     protected function initialize(InputInterface $input, OutputInterface $output)
     {
         $this->io = new SymfonyStyle($input, $output);
+
+        $configs = Yaml::parse(file_get_contents(__DIR__.'/../../../.sonata-project.yml'));
+        $processor = new Processor();
+        $this->configs = $processor->processConfiguration(new Configuration(), array('sonata' => $configs));
+
+        $this->packagistClient = new \Packagist\Api\Client();
+
+        $this->githubClient = new \Github\Client(
+            new \Github\HttpClient\CachedHttpClient(array(
+                'cache_dir' => sys_get_temp_dir().'/github-api-cache',
```

Comment:
A good practice I found by suffering from crowded `/tmp/` directories on my servers is to also include the name of the project that generates stuff and sometimes does not clean afterwards. So using `/sonata-dev-kit/github-api-cache` would be better IMO